airbyte-integrations/connectors/source-pipedrive/source_pipedrive/manifest.yaml

```diff
@@ -411,3 +411,15 @@ spec:
         examples:
           - "2017-01-25 00:00:00Z"
         type: string
+      num_workers:
+        title: Number of concurrent workers
+        description: Number of concurrent workers to be used in the sync
```

Comment:
Please add a link to the rate limit documentation so users know where to look to set this values. See example for Asana:

https://github.com/airbytehq/airbyte/pull/48712/files#diff-c1cd8d26d6d94022c0e41970a2798febd54173393b1aebe977a7646ec015ce11R2609-R2612

Reply:
updated